variables.tf

```diff
@@ -0,0 +1,29 @@
+variable "prefix" {
+    description = "The prefix used for all resources in this environment"
+    default = "ke1"
+}
+variable "SECRET_KEY"{
+    description = ""
+    default = "secret-key"
```

Comment:
This is fine for the exercise, but in reality you wouldn't want a default for this. You need to make sure you have a secure value for this - if it were leaked people could fake their login session cookie.